cluster/cluster.go

```diff
@@ -153,7 +153,7 @@ func (c *Cluster) createInner(ctx context.Context) error {
 		info = toInfo(c)
 	}
 
-	if c.Status == Updating || c.Status == Running || c.Status == PostCheck {
+	if c.Status == Updating || c.Status == Running || c.Status == PostCheck || c.RootCACert != "" {
```

Comment:
Can we introduce an Init status? So c.Status is set to Init as soon as the cert is generated. This way we can be more inline with the state checks.